service/operator/service.go

```diff
@@ -0,0 +1,138 @@
+package operator
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/giantswarm/azuretpr"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/tpr"
+	"github.com/spf13/viper"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+
+	"github.com/giantswarm/azure-operator/flag"
+)
+
+const (
+	TPRVersion     = "v1"
+	TPRDescription = "Managed Kubernetes clusters in Giantnetes running on Azure"
```

Comment:
This is already defined in the TPR repo and should be used from there. See https://github.com/giantswarm/azuretpr/blob/32a7ab664cbb9036f6e1707cbdb7ce1b2d1593b8/meta.go.